### PR TITLE
CONN-92 Remove NHINC_PROPERTIES_DIR Env Variable

### PR DESCRIPTION
**Pull request by jsmith2**
- Removed all references to NHINC_PROPERTIES_DIR
- validation_testing_driver.xml no longer used, so removed.
- PropertyAccessorFileUtilities was formatted, so whole file appears changed, only review the checkPropertyFileDir() for changes.

### CODE REVIEW

File: Product/Production/Services/HIEMCore/src/main/java/gov/hhs/fha/nhinc/hiem/configuration/topicconfiguration/TopicConfigurationManager.java [hunk 1]
```diff
@@ -161,7 +160,6 @@ private static void loadTopicConfigurations() throws ConfigurationException {
         topicConfigurations = TopicConfigurationsXML.deserialize(sTopicConfigXml);
 
         if (topicConfigurations != null) {
-            m_bInternalLoaded = true;
```
Comment (msweaver) [inline Product/Production/Services/HIEMCore/src/main/java/gov/hhs/fha/nhinc/hiem/configuration/topicconfiguration/TopicConfigurationManager.java:164]: This seems weird. Maybe change the if statement to but == null, add the log statement to the if part and remove the else?

File: Product/Install/validation_testing_driver.xml [hunk 2]
```diff
@@ -1,446 +0,0 @@
```
Comment (msweaver) [inline Product/Install/validation_testing_driver.xml:-1]: Why delete this file?

Comment (jsmith2) [inline Product/Install/validation_testing_driver.xml:-1]: This was used for the old CI pipeline and has been subsequently replaced by maven.

Comment (msweaver) [inline Product/Install/validation_testing_driver.xml:-1]: ok makes sense.

File: Product/Production/Services/HIEMCore/src/main/java/gov/hhs/fha/nhinc/hiem/configuration/topicconfiguration/TopicConfigurationManager.java (hunk 1 above)
Comment (jsmith2) [inline Product/Production/Services/HIEMCore/src/main/java/gov/hhs/fha/nhinc/hiem/configuration/topicconfiguration/TopicConfigurationManager.java:164]: This was an unintended consequence of removing a class variable marked as unused.  I'll update that to be a if(== null) and get rid of the else.

**Pull request merged**